apps/common-app/src/examples/LayoutAnimations/ListItemLayoutAnimation.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { memo, useCallback, useState } from 'react';
+import type { ListRenderItem } from 'react-native';
+import {
+  Dimensions,
+  Pressable,
+  SafeAreaView,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+} from 'react-native';
+import Animated, {
+  LayoutAnimationConfig,
+  LinearTransition,
+} from 'react-native-reanimated';
+
+const ITEMS = ['Item 1', 'Item 2', 'Item 3', 'Item 4', 'Item 5'];
+
+type ListItemProps = {
+  id: string;
+  text: string;
+  onPress: (id: string) => void;
+};
+
+const ListItem = memo(function ({ id, text, onPress }: ListItemProps) {
+  return (
+    <Pressable onPress={() => onPress(id)} style={styles.listItem}>
+      <Text style={styles.itemText}>{text}</Text>
+    </Pressable>
+  );
+});
+
+export default function ListItemLayoutAnimation() {
+  const [items, setItems] = useState(ITEMS);
+
+  const handlePress = useCallback((id: string) => {
+    setItems((prevItems) => prevItems.filter((item) => item !== id));
+  }, []);
+
+  const renderItem = useCallback<ListRenderItem<string>>(
+    ({ item }) => <ListItem id={item} text={item} onPress={handlePress} />,
+    [handlePress]
+  );
+
+  const findItemName = useCallback(() => {
+    let i = 1;
+    while (items.includes(`Item ${i}`)) {
+      i++;
+    }
+    return `Item ${i}`;
+  }, [items]);
+
+  const reorderItems = useCallback(() => {
+    setItems((prevItems) => {
+      const newItems = [...prevItems];
+      newItems.sort(() => Math.random() - 0.5);
+      return newItems;
+    });
+  }, []);
+
+  const resetOrder = useCallback(() => {
+    setItems((prevItems) => {
+      const newItems = [...prevItems];
+      newItems.sort((a, b) => {
+        const aNum = parseInt(a.match(/\d+$/)![0], 10);
+        const bNum = parseInt(b.match(/\d+$/)![0], 10);
+        return aNum - bNum;
+      });
+      return newItems;
+    });
+  }, []);
+
+  const transition = LinearTransition;
+
+  return (
+    // Skip initial entering animation
```

Comment:
A line below tells exactly the same.
```suggestion
    // Skip initial entering animation
```